tools/shared.py

```diff
@@ -2756,6 +2757,15 @@ def wasm2js(js_file, wasm_file, opt_level, minify_whitespace, use_closure_compil
       f.write(all_js)
     return js_file
 
+  @staticmethod
+  def emit_debug_on_side(wasm_file):
+    # extract the DWARF info from the main file, and leave the wasm with
+    # debug into as a file on the side
+    # TODO: use strip directly; for now use objcopy
+    wasm_file_with_dwarf = wasm_file + '.debug.wasm'
+    shutil.copyfile(wasm_file, wasm_file_with_dwarf)
```

Comment:
even faster would be `shutil.move()` instead of a copy, followed by `objcopy --remove-section=.debug* wasm_file_with_dwarf wasm_file`